src/core/positioning/nmeagnssreceiver.cpp

```diff
@@ -85,3 +105,61 @@ void NmeaGnssReceiver::handleStopLogging()
 {
   mLogFile.close();
 }
+
+void NmeaGnssReceiver::handleIMUCorrection( const QString &sentence )
+{
+  static const int PARAMETER_STATUS = 19;
+
+  static const int IMU_KQGEO_STATUS_OK = 1026;
+  static const int IMU_KQGEO_STATUS_OK_NEW = 1967106;
+
+  QStringList parameters = sentence.split( ',' );
+  if ( parameters.size() < PARAMETER_STATUS )
+    return;
+
+  // Parse Status
+  bool ok = false;
+  const int status = parameters[PARAMETER_STATUS].toInt( &ok );
+  if ( ok == false )
+  {
+    mIMUCorrectedPosition.valid = false;
+    return;
+  }
+
+  if ( status != IMU_KQGEO_STATUS_OK && status != IMU_KQGEO_STATUS_OK_NEW )
+  {
+    mIMUCorrectedPosition.valid = false;
+    return;
+  }
+
+  mIMUCorrectedPosition.valid = true;
+
+  // Parse other parameters
+
+  QDateTime utcDateTime = QDateTime::currentDateTime();
+  QTime time = QTime::fromString( parameters[1], "hhmmss.zzz" );
+  if ( time.isValid() )
+    utcDateTime.setTime( time );
```

Comment:
This is not being used anymore as far as I can see, did you mean to then set the mIMUCorrectedPosition date time member with it?

Reply:
Yes thanks